syntax = "proto3";

message E3SensorData {
  string sensor_name = 1;
  uint64 num_vehicles = 2;
}

message TrafficLightState {
  string traffic_light_name = 1;
  string state = 2;
}
